src/pybind/mgr/dashboard/frontend/src/app/ceph/smb/smb-cluster-form/smb-cluster-form.component.html

```diff
@@ -0,0 +1,343 @@
+            type="multi"
+            selectionFeedback="top-after-reopen"
+            label="Label"
+            for="label"
+            name="label"
+            formControlName="label"
+            id="label"
+            placeholder="Select labels..."
+            [appendInline]="true"
+            [items]="data.labels"
+            i18n-placeholder
+            (selected)="multiSelector($event, 'label')"
+            [invalid]="smbForm.controls.label.invalid && smbForm.controls.label.dirty"
+            [invalidText]="labelError"
+            cdRequiredField="Label"
+            i18n
+          >
+            <cds-dropdown-list></cds-dropdown-list>
+          </cds-combo-box>
+          <ng-template #labelError>
+            <span
+              class="invalid-feedback"
+              *ngIf="smbForm.showError('label', formDir, 'required')"
+              i18n
+              >This field is required.</span
+            >
+          </ng-template>
+        </div>
+
+        <!-- Hosts -->
+        <div *ngIf="smbForm.controls.placement.value === 'hosts'"
+             class="form-item">
+          <cds-combo-box
+            type="multi"
+            selectionFeedback="top-after-reopen"
+            label="Hosts"
+            for="hosts"
+            name="hosts"
+            formControlName="hosts"
+            id="hosts"
+            placeholder="Select hosts..."
+            i18n-placeholder
+            [appendInline]="true"
+            [items]="data.hosts"
+            (selected)="multiSelector($event, 'hosts')"
+            i18n
+          >
+            <cds-dropdown-list></cds-dropdown-list>
+          </cds-combo-box>
+        </div>
+      </ng-container>
+    </ng-container>
+
+    <div class="form-item">
+      <cds-number
+        [id]="'count'"
+        [name]="'count'"
+        [formControlName]="'count'"
+        [label]="'Count'"
+        [min]="1"
+        i18n
+      ></cds-number>
+    </div>
+
+    <!-- Clustering -->
+    <div class="form-item">
+      <cds-select
+        formControlName="clustering"
+        name="clustering"
+        for="clustering"
+        label="Clustering"
+        id="clustering"
+        i18n
+      >
+        <option *ngFor="let data of allClustering">{{ data | upperFirst }}</option>
+      </cds-select>
+      <cd-help-text>
+        Control if a cluster abstraction actually uses Samba’s clustering mechanism.
+      </cd-help-text>
+    </div>
+
+    <!-- Custom DNS -->
+    <ng-container formArrayName="custom_dns"
+                  *ngFor="let dns of custom_dns.controls; index as i">
+      <div cdsRow
+           class="form-item form-item-append">
+        <div cdsCol
+             [columnNumbers]="{ lg: 8 }">
+          <input cdsText
+                 [formControlName]="i"
+                 placeholder="Custom DNS"
+                 modal-primary-focus />
+        </div>
+        <div cdsCol
+             [columnNumbers]="{ lg: 1 }"
+             class="item-action-btn">
```

Comment:
i've seen this css class, but there is no mention of what it is neither i can find it in css
```suggestion
             class="item-action-btn">
```